src/main/java/com/znsio/teswiz/runner/Driver.java

```diff
@@ -464,4 +469,27 @@ public void uploadFileInBrowser(String filePath, By locator) {
                     e);
         }
     }
+
+    /**
+     * This method injects the media to browserstack to perform,
+     * image scanning eg: QRcode,barcode etc
+     * @param uploadFileURL
+     */
+    public void injectMediaToBrowserstackDevice(String uploadFileURL) {
+        String cloudName = Setup.getFromConfigs(CLOUD_NAME);
+        try {
+            if (Setup.getPlatform().equals(Platform.android) && cloudName == "browserstack") {
+                String cloudUser = Setup.getFromConfigs(Setup.CLOUD_USER);
+                String cloudKey = Setup.getFromConfigs(CLOUD_KEY);
+                BrowserStackImageInjection.injectMediaToDriver(uploadFileURL, ((AppiumDriver) driver), cloudUser, cloudKey, cloudName);
```

Comment:
why do you need to pass cloudName?

Reply:
changed the method signature , not taking care in the internal method